email-smtp/src/main/java/io/commercetools/sunrise/email/smtp/SmtpAuthEmailSender.java

```diff
@@ -1,8 +1,32 @@
 package io.commercetools.sunrise.email.smtp;
 
 import io.commercetools.sunrise.email.EmailSender;
+import io.commercetools.sunrise.email.MessageAuthor;
 
+import javax.annotation.Nonnull;
+import javax.mail.internet.MimeMessage;
+import java.util.Properties;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+/**
+ * An e-mail sender that sends e-mail over SMTP using the default implementation of the Java Mail API.
+ */
 public class SmtpAuthEmailSender implements EmailSender {
 
-    // TODO
+    protected SmtpAuthEmailSender(@Nonnull final Properties properties) {
+    }
+
+    public static
+    @Nonnull
+    SmtpAuthEmailSender of(@Nonnull final Properties properties) {
+        return new SmtpAuthEmailSender(properties);
+    }
+
+    @Override
```

Comment:
actually I would prefer that visibility specifiers and annotations are not mixed and the annotations comes first with every annotation in one line